htdocs/web_portal/views/project/view_project.php

```diff
@@ -50,34 +50,32 @@
 
         <?php if ($ngiCount != 0): ?>
             <table id="ngisTable" class="table table-striped table-condensed tablesorter">
-        <thead>
-            <tr>
-            <th></th>
-            <th>Name</th>
-            <th>Description</th>
-            </tr>
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th>Name</th>
+                        <th>Description</th>
+                    </tr>
                 </thead>
-        <tbody>
-                <?php
-                //$num = 2;
-                foreach($params['NGIs'] as $ngi) {
-                ?>
-                <tr>
-            <td>
-            <img class="flag" src="<?php echo \GocContextPath::getPath()?>img/ngi/<?php xecho($ngi->getName()) ?>.jpg">
-            </td>
-                    <td>
-            <a href="index.php?Page_Type=NGI&id=<?php echo $ngi->getId() ?>">
-                <?php xecho($ngi->getName()); ?>
-            </a>
-                    </td>
-                    <td><?php xecho($ngi->getDescription()) ?></td>
-                </tr>
-                <?php
-                    //if($num == 1) { $num = 2; } else { $num = 1; }
-                } // End of the foreach loop iterating over SEs
-                ?>
-        </tbody>
+                <tbody>
+                    <?php
+                    foreach($params['NGIs'] as $ngi) {
+                    ?>
+                        <tr>
+                            <td>
+                                <img class="flag" src="<?php echo \GocContextPath::getPath()?>img/ngi/<?php xecho($ngi->getName()) ?>.jpg">
+                            </td>
+                            <td>
+                                <a href="index.php?Page_Type=NGI&id=<?php echo $ngi->getId() ?>"><?php xecho($ngi->getName()); ?></a>
```

Comment:
Will this conflict with the escape fixes?

Reply:
Yeees? Maybe?

Reply:
Turns out nope, doesn't conflict!